.github/workflows/breaking-change-warning.yml

```diff
@@ -0,0 +1,50 @@
+name: Breaking Change Warning
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches:
+      - main
+      - release/*
+      - next
+
+jobs:
+  check_broken_types:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get changed files
+        id: get-files
+        run: |
+          echo "::set-output name=files::$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | xargs)"
+
+      - name: Check for changes to broken types
+        id: check
```

Comment:
Is there a reason to split these two steps? Just personal preference, I think I would have consolidated them, but it's also fine like this.

Reply:
Good catch! This was a leftover from a previous iteration that I forgot to consolidate